### DISCUSSION TITLE
Why wen i try to sent an email with HTML format the text don't set in the email, but the img do appears and also the format?

### DISCUSSION THREAD

**Discussion by IvanCoto** [PHPMailer/PHPMailer#2729] [Q&A]
I am trying to send an html formatted email but when I send it only the formatting and images are displayed, however the text is not displayed. Does anyone know how to fix this?  

![image](https://user-images.githubusercontent.com/84398566/178128991-858d3db9-67ba-442f-b78f-028ba3027a93.png)

Basically this is an example: 

`<td class="text-header" style="color:#999999; font-family: "Noto Sans", Arial,sans-serif; font-size:12px; line-height:16px; text-align:right; text-transform:uppercase;"><multiline><webversion class="link2" style="color:#999999; text-decoration:none;"><a href="https://www.ci-technologycr.com/">Open in browser</a></webversion></multiline></td>`

This is the email: 
![image](https://user-images.githubusercontent.com/84398566/178129018-959cf86f-7937-4a18-a67b-d87629c3cb7a.png)

**Comment by Synchro** [comment]
You're using made-up tags that the client is ignoring. There is no such tag as `<webversion>`
